src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/results.html.twig

```diff
@@ -19,9 +19,24 @@
                 {% for elements in tokens %}
                     <tr>
                         <td><a href="{{ path('_profiler', { 'token': elements.token }) }}">{{ elements.token }}</a></td>
-                        <td>{{ elements.ip }}</td>
-                        <td>{{ elements.method }}</td>
-                        <td>{{ elements.url }}</td>
+                        <td>
+                            {{ elements.ip }}
+                            <a href="{{ path('_profiler_search_results', app.request.query.all|merge({'ip': elements.ip, 'token': elements.token})) }}">
```

Comment:
Without that no search results will be shown at all. The filtering does not happen on the token, but the token is part of search result URI:

    https://symfony.dev/_profiler/01d9c6/search/results?ip=&limit=10&url=https%3A//symfony.dev/filter-link